tests/Functional/Fixtures/symfony/app/config/config.yml

```diff
@@ -8,24 +8,24 @@ services:
 framework:
     secret: 'azertyuiop'
     router:
-        resource: '%kernel.root_dir%/config/routing.yml'
+        resource: '%kernel.project_dir%/app/config/routing.yml'
         strict_requirements: ~
     session:
         # http://symfony.com/doc/current/reference/configuration/framework.html#handler-id
         handler_id:  session.handler.native_file
-        save_path:   "%kernel.root_dir%/../var/sessions/%kernel.environment%"
+        save_path:   "%kernel.project_dir%/app/var/sessions/%kernel.environment%"
 
 doctrine:
     dbal:
         driver:   pdo_sqlite
-        path:     "%kernel.root_dir%/data/data.sqlite"
+        path:     "%kernel.project_dir%/app/data/data.sqlite"
```

Comment:
`app` does not exists anymore in SF 4.X
Let's move this to ` "%kernel.project_dir%/data/data.sqlite"`

Reply:
If we do that, do we have to update the folder structure for all the SF app (which is in an `app` folder)?

Reply:
Could be better indeed, but could be left also. It's up to you

Reply:
Because this is a big change, I created an issue to fix the old architecture -> #41